dev-support/hbase-personality.sh

```diff
@@ -146,7 +146,7 @@ function personality_modules
   fi
 
   if [[ -n "${HADOOP_PROFILE}" ]]; then
-    extra="${extra} -Phadoop-${HADOOP_PROFILE}"
+    extra="${extra} -Dhadoop.profile=${HADOOP_PROFILE}"
```

Comment:
Also need the branch check here.

Reply:
My assumption was that we could do this when we invoke yetus and then not have to sully the personality itself.

I guess there is an expectation that people can invoke yetus by hand, too. Makes me wonder if it's easier to just do it all in the personality then.